index.html

```diff
@@ -195,15 +195,14 @@
                   "url": "https:\/\/keboola-direct.zendesk.com"
                 }
               },
-              "adminEmails": ["martin@keboola.com"],
               "goodDataTokens": {
                 "developer": "developer",
                 "production": "production"
               },
               "admin": {
```

Comment:
```suggestion
               "admin": params.token.admin
```
Tu staci vlastne dat len `params.token.admin`, tie info tam su.

Reply:
V tom případě nám to nemusí server renderovat při startu.

Reply:
Asi nerozumiem. Co nemusi server renderovat pri starte? Tu sa myslim nic nerenderuje, je to len predanie tokenu z verifyToken callu

Reply:
Na to jsem myslím už koukal někdy. Verify vrací jinou strunturu dat. I podle API dokumentace: https://keboola.docs.apiary.io/#reference/tokens-and-permissions/token-verification/token-verification

Reply:
Aha tak koukám to tam je. A to jsem myslel že jsem fakt už někdy koukal, divné.

Reply:
Nevidím tam akorát `email` v tom `admin` objektu.

Reply:
`params.token.admin` nejde použít, obsah se liší. Konkrétně klíče `email`, `profileImageUrl` a `isOrganizationMember`. Liší se i pole features - v sapiToken je prázdné.

Reply:
Tady vůbec není, že by v response byl `admin`: https://keboola.docs.apiary.io/#reference/tokens-and-permissions/token-verification/token-verification

(nebo blbě hledám?)

Reply:
jestli v te doc neco chybi nebo napoak prebyva nez co to vraci ve skutecnosti, tak pls issue https://github.com/keboola/storage-api-php-client 🙏

Reply:
Uff, toto je nejake nedorozumenie. Ved index.html je devel cast, mozem si tam natlacit co chcem.  `params.token.admin` urcite ide pouzit, skusal som to ... profileImageUrl tam bola nejaka dummy.

Reply:
@ujovlado `index.html` by imo měl mít stejnou strukturu, jakou renderuje server při startu do `kbcApp.start()`.

Reply:
To samozrejme nejde, lebo cez storage api neziskas vsetky info, ktore potrebujes a na nieco tam budes musiet mat "dummy text", co bolo vidno aj tu. Takze by som to neriesil.

Reply:
Asi bych nakonec nedělal žádnou úpravu tam. Nepřijde mi to k lepšímu.
Takto když tam nebudu mít třeba email tak ho nebudu mít v hlavičce u jména, třeba u jobů jak mám predefined hledání tak tam místo emailu je undefined apod. Radši ať tam je ten example.

Reply:
Tak zrovna ten email by si si tam mohol doplnit a ostatne info by bolo z verify response

Reply:
Jop nebo něco tak namíchané no :)

Reply:
@natocTo pri tych joboch ti to ale ani teraz nefunguje, lebo ten example email tam nema nikto :)

Reply:
jj to vím :) vím že mě to taky jednou nachytalo dříve

Reply:
Měnil jsem to hlavně proto, že je tam `admin.id: 1`, kvůli tomu se nerenderoval `yours` label a neřadilo to. A jak vidím, asi to nebude jediný case, kde to něco ovlivní. Proto bych to dal do konfigurace, jak to mám navržené.

Reply:
Ale to je blbost mat v configu dummy data, ktore sa daju ziskat z api. To si tam kazdy z nas da svoje idcko?